docs/ml-guide.md

```diff
@@ -455,6 +459,17 @@ val pipeline = new Pipeline()
 // Fit the pipeline to training documents.
 val model = pipeline.fit(training)
 
+// now we can optionally save the fitted pipeline to disk
+model.save("/tmp/spark-logistic-regression-model")
+
+// we can also save this unfit pipeline to disk
+pipeline.save("/tmp/unfit-lr-model")
+
+// and load it back in during production
+val sameModel = Pipeline.load("/tmp/spark-logistic-regression-model")
+// or equivalently
```

Comment:
Remove this and next line
> I like the idea in this & the next line, but I'd remove this since it will cause problems if users blindly copy and paste the code into a spark shell.